### DISCUSSION TITLE
A Few Simple Ideas

### DISCUSSION THREAD

**Discussion by JT-Apps** [v8hid/infinite-zoom-automatic1111-webui#15] [Ideas]
Would love to see: simple rotation option, simple drag n drop audio track option so we can throw on a .mp3/.aac for a small soundtrack on the export.

A bit more creative but a really cool thing would be (simple, ha) ChatGPT API key plugin where I can just have it fill in both frame number + prompt row, either entirely from nothing or perhaps just 'picking up' from whatever your first prompt says. Haven't seen too many straightforward ChatGPT/OpenAI API key implementations in SD plugins but could work so well.

**Comment by DawgZter** [comment]
+1 to all of this!

**Reply by Oncorporation** [reply]
I almost added the music track last week.
